repair/repair.cc

```diff
@@ -419,13 +422,17 @@ size_t tracker::nr_running_repair_jobs() {
     return count;
 }
 
+bool tracker::is_aborted(const utils::UUID& uuid) {
+    return _aborted_pending_repairs.contains(uuid);
+}
+
 void tracker::abort_all_repairs() {
-    size_t count = nr_running_repair_jobs();
+    _aborted_pending_repairs = _pending_repairs;
```

Comment:
Is `tracker::done()` called also when repair is aborted with an exception?

Reply:
Yes.